proxy/src/write.rs

```diff
@@ -527,7 +527,8 @@ impl Proxy {
                 Err(e) => {
                     // TODO: remove this logic.
                     // Refer to https://github.com/CeresDB/ceresdb/issues/1248.
-                    if e.error_message().contains("No field named") {
+                    if e.error_message().contains("decode row group payload") {
```

Comment:
Define a method `need_evict_table() bool` to include all this error message required to evict table.